vmdb/app/models/ems_refresh/parsers/openstack_infra.rb

```diff
@@ -111,11 +111,71 @@ def load_hosts
         end
       end
 
+      def parse_lscpu(lscpu_data)
+        parsed_data = {}
+
+        lscpu_data.each_line do |line|
+          parts = line.split(/:/)
+          value = parts[1].try(:chomp).try(:strip)
+          key = case parts[0]
+                when 'Socket(s)'          then 'numvcpus'
+                when 'Core(s) per socket' then 'cores_per_socket'
+                when 'CPU MHz'            then 'cpu_speed'
+                when 'Model name'         then 'cpu_type'
+                end
+          parsed_data[key] = value
+        end
+        parsed_data
+      end
+
+      def parse_dmidecode(dmidecode_data)
+        parsed_data = {}
+
+        dmidecode_data.each_line do |line|
+          parts = line.split(/:/)
+          value = parts[1].try(:chomp).try(:strip)
+          key = case parts[0].strip
+                when 'Manufacturer'  then 'manufacturer'
+                when 'Product Name'  then 'model'
+                when 'Version'       then 'guest_os_full_name'
+                when 'Family'        then 'guest_os'
+                when 'Serial Number' then 'service_tag'
+                end
+          parsed_data[key] = value
+        end
+        parsed_data
+      end
+
+      def get_extra_host_attributes!(host, hostname)
+        # TODO(lsmola) in RHOS7 we can get this stuff from ironic, but now we need to hack it. This will need to
+        # obtain host from DB, so it requires already existing host. So it will be always filled second refresh.
+        # Once RHOS7 Ironic is here, we need to revisit indexes in extra data, that is changing a lot, then delete this
+        host_for_ssh = HostOpenstackInfra.new(:hostname => hostname)
+        host_for_ssh.ext_management_system = @ems
+
+        begin
+          host_for_ssh.connect_ssh do |ssu|
```

Comment:
@blomquisg this stuff should be just temporary, in next release we will get this from Ironic API. We need the Kilo Ironic for this.

Except the operation system version, which we don't need that much I guess. It could be obtained as part of fleecing.

So I guess this should be ok.